components/atom/toast/src/index.js

```diff
@@ -0,0 +1,149 @@
+import React, {useState, useEffect, useRef, useCallback} from 'react'
+import PropTypes from 'prop-types'
+import IconClose from '@schibstedspain/sui-svgiconset/lib/Close'
+import cx from 'classnames'
+import {
+  AUTO_CLOSE_TIMES,
+  BASE_CLASS,
+  EFFECT_DELAY,
+  MARGINS,
+  POSITIONS,
+  SIZES
+} from './config'
+
+function AtomToast({
+  autoClose = true,
+  autoCloseTime = AUTO_CLOSE_TIMES.medium,
+  children,
+  crossToClose = true,
+  effect = true,
+  globalClose = false,
+  iconClose = <IconClose />,
```

Comment:
I think we decided to avoid using  default icon components in order to avoid loading twice (the default one and the passed). So, I would add  this as required prop so people could add their custom crosss for every vertical.

Reply:
oh.. good one.
I will adjust the Specs in jira to reflect this.